packages/matter.js/src/common/Scanner.ts

```diff
@@ -51,6 +31,43 @@ export type CommissionableDevice = {
 
     /** Pairing instructions */
     PI?: string;
+
+    /** Sleep Idle Interval */
+    SII?: number;
```

Comment:
Is this sufficiently camel-cased?

Reply:
I choose to be 1:1 to the MDNS txt key. That's why all uppercase. Wdyt?

Reply:
Yeah all good, was just kidding :)